pkg/oci/load_balancer_util.go

```diff
@@ -505,3 +507,45 @@ func sortAndCombineActions(backendSetActions []Action, listenerActions []Action)
 	})
 	return actions
 }
+
+func getLoadBalancerSourceRanges(config *Config, service *api.Service) ([]string, error) {
```

Comment:
Just pass in `config.LoadBalancer.DefaultSourceCIDRs` rather than the full config struct.